### DISCUSSION TITLE
Occupation

### DISCUSSION THREAD

**Discussion by realkibou** [digitalengineering/airbnb-scraper#20] [Ideas]
Hi there,

I hope you are fine, thank you for the bot. :). Do you think it is possible to also get the occupation rate? Let's say I crawl at the end of the month, and look back on that month, to see which dates have been blocked/booked?

Cheers!

**Comment by digitalengineering** [comment]
Hi @realkibou, 

Thank you for the question, it's an interesting one. As you point out, it would require re-crawling a pre-defined list of rental properties periodically. However, with a pre-defined list of rental properties, I think a relatively short vanilla python script could accomplish this goal without needing `scrapy`.

While it's certainly possible to build this functionality directly into `airbnb-scraper`, I can't currently think of any way to do it that wouldn't involve a fair bit of work.

**Reply by realkibou** [reply]
I am fairly new to Scrapy... I spend some time, but could not get it going.

Basically the scraper would just go to this URL `https://www.airbnb.com/rooms/35594771?#availability-calendar`
Look for the (multiple) `tbody`s for the `table` with the `role=presentation` and go through each `tr` and then `td` to export the `aria-label` which would give you for example "Not available Thursday, June 9, 2022". 

Then throw this out as a separate CSV with the listing's url in column A and then the `aria-label`'s content in each following column. One listing per row.

Do you think you can throw that "quick and dirty" together?
